.github/workflows/citgm.yml

```diff
@@ -0,0 +1,173 @@
+name: CITGM
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  npm-cache:
+    name: Cache Node Modules for Fastify
+    if: ${{ github.event.label.name == 'citgm-core-plugins' }}
+    runs-on: ${{ matrix.os }}
+    permissions:
+      contents: read
+    strategy:
+      matrix:
+        node-version: ['lts/*']
```

Comment:
let’s name those. Having a star is too unpredictable.

Reply:
Set it to 20, lets get the citgm green and then decide which node versions we want